kedro/templates/project/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -1,14 +1,7 @@
-black~=22.0
-flake8>=3.7.9, <5.0
 ipython>=7.31.1, <8.0; python_version < '3.8'
 ipython~=8.10; python_version >= '3.8'
-isort~=5.0
 jupyter~=1.0
 jupyterlab_server>=2.11.1, <2.16.0
 jupyterlab~=3.0, <3.6.0
 kedro~={{ cookiecutter.kedro_version }}
 kedro-telemetry~=0.2.0
-nbstripout~=0.4
```

Comment:
Noticed this is used by the e2e test "Activate_nbstripout target in new project", which is not related to documentation, and so `nbstripout` should be present in the default bare-bones requirements file

Reply:
The `activate_nbstripout` command will be removed in `0.19.0`, so I'd suggest just removing it here already if it doesn't interfere with tests.

Reply:
This can now be removed because the commands + tests were removed by Nok in https://github.com/kedro-org/kedro/pull/3065